coral-trino/src/test/java/com/linkedin/coral/trino/trino2rel/TrinoToRelConverterTest.java

```diff
@@ -25,28 +25,34 @@
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
+import com.linkedin.coral.hive.hive2rel.functions.StaticHiveFunctionRegistry;
+
 import static com.linkedin.coral.trino.trino2rel.ToRelTestUtils.*;
-import static com.linkedin.coral.trino.trino2rel.TrinoCalciteUDFMapUtils.*;
+import static com.linkedin.coral.trino.trino2rel.TrinoCalciteTransformerMapUtils.*;
 import static org.apache.calcite.sql.type.OperandTypes.*;
-import static org.testng.Assert.*;
+import static org.testng.AssertJUnit.assertEquals;
 
 
 public class TrinoToRelConverterTest {
   private static HiveConf conf;
 
   @BeforeClass
   public void beforeClass() throws HiveException, IOException, MetaException {
+    // Simulating a Coral environment where "foo" exists
```

Comment:
Make sure we register UDFs before instantiating TrinoToRelConverter so they are included in the registry